pymodm/fields.py

```diff
@@ -1137,6 +1158,36 @@ def to_mongo(self, value):
         return [self._model_to_document(doc) for doc in value]
 
 
+class EmbeddedDocumentListField(_EmbeddedDocumentListFieldBase):
+    """**DEPRECATED**: A field that stores a list of documents within a document.
+
+    All documents in the list must be of the same type.
+    """
+    def __init__(self, model, verbose_name=None, mongo_name=None, **kwargs):
+        warnings.warn('EmbeddedDocumentListField is deprecated. Use '
+        'EmbeddedModelListField instead.', DeprecationWarning, stacklevel=2)
+        kwargs.setdefault('default', list)
+        super(EmbeddedDocumentListField, self).__init__(
+            model=model,
+            verbose_name=verbose_name,
+            mongo_name=mongo_name,
+            **kwargs)
+
+
+class EmbeddedModelListField(_EmbeddedDocumentListFieldBase):
+    """A field that stores a list of documents within a document.
+
+    All documents in the list must be of the same type.
+    """
+    def __init__(self, model, verbose_name=None, mongo_name=None, **kwargs):
+        kwargs.setdefault('default', list)
+        super(EmbeddedModelListField, self).__init__(
+            model=model,
+            verbose_name=verbose_name,
+            mongo_name=mongo_name,
+            **kwargs)
```

Comment:
Do we even need the __init__ method here? It is the same thing as the base class so I think we can remove it entirely.